sled-agent/src/rack_setup/service.rs

```diff
@@ -130,8 +131,27 @@ use std::iter;
 use std::net::{Ipv6Addr, SocketAddrV6};
 use std::time::Duration;
 use thiserror::Error;
+use tokio::sync::watch;
 use uuid::Uuid;
 
+/// For tracking the current RSS step and sending notifications about it.
+pub struct RssProgress {
+    step: RssStep,
```

Comment:
I don't think this field is used - could be removed? At that point I'm not sure this struct is useful except as a way to renamed `.send()` to `.update()` (which might be worth it - `.send()` being fallible is weird, although there are options there I'll leave in a separate comment).

Reply:
removed